src/LSST/system/TelemetryItemDefs.h

```diff
@@ -0,0 +1,606 @@
+/*
+ *  This file is part of LSST M2 support system package.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+#ifndef LSST_M2CELLCPP_SYSTEM_TELEMETRYITEMDEFS_H
+#define LSST_M2CELLCPP_SYSTEM_TELEMETRYITEMDEFS_H
+
+// System headers
+
+// Third party headers
+#include "nlohmann/json.hpp"
+
+// project headers
+#include "system/TelemetryItem.h"
+
+namespace LSST {
+namespace m2cellcpp {
+namespace system {
+
+/// common elements between powerStatus and powerStatusRaw
+/// Unit tests in tests/test_TelemetryCom
+class TItemPowerStatusBase : public TelemetryItem {
+public:
+    TItemPowerStatusBase(std::string const& id) : TelemetryItem(id) {}
+
+    virtual ~TItemPowerStatusBase() = default;
+
+    /// Return reference to `_motorVoltage`.
```

Comment:
Please help to put the unit for all the physical amount (`Volt`, `Amp`, `N`, `N m`, `degC`, deg, etc.). You can use the following as a reference:
https://github.com/lsst-ts/ts_xml/blob/develop/python/lsst/ts/xml/data/sal_interfaces/MTM2/MTM2_Telemetry.xml

Reply:
I'll do that for everything I can find and leave notes for those where I'm not certain.